chore(www): Unified Logs open beta blog post (#47645)
supabase/supabase: apps/www/_blog/2026-07-14-unified-logs-open-beta.mdx

---
title: 'Unified Logs is now in open beta'
description: 'One searchable view for logs across every Supabase service, with live tail, filtering, and a timeline.'
author: jordi, kemal_y, steven_eubank
date: '2026-07-14'
categories:
  - product
tags:
  - logs
  - observability
imgSocial: 'unified-logs-open-beta/og.jpg'
imgThumb: 'unified-logs-open-beta/thumb.jpg'
toc_depth: 2
---

Today Unified Logs enters open beta. It brings logs from every service in your project into one searchable view: the API gateway, Postgres, Auth, Storage, PostgREST, Realtime, and the connection poolers.

Before this, debugging a single request meant jumping between separate log pages. You checked the API gateway logs, then Postgres logs, then Auth logs, each with its own filters and its own tab. Correlating a request across services was slow, and by the time you found the right page the context was gone.

Unified Logs puts all of that in one place.

## One stream across your services

> Open Logs in the dashboard and you see every event as it happens, newest first. Each row shows the timestamp, the service, the method, the path, and the status. Postgres connection events sit next to API requests sit next to Auth events, in the same table.

Reviewer: ```suggestion
Open Logs in the dashboard and you see every event as it happens, newest first. Each row shows the timestamp, the service, the method, the path, and the status. One row might be an API request, the next a Postgres connection, the next an Auth event, all in the same table.
```

## Filter and search

The filter bar at the top understands the shape of your logs. Filter by log type, level, status, method, or pathname, and combine those filters however you need. You can also run a free-text search across the event message to find a specific request.

Filters live in the URL, so you can share a link to exactly what you are looking at with a teammate.

## Live tail

Click Live to stream new events into the table as they arrive. This is the fastest way to watch a deploy, reproduce a bug, or confirm that traffic is reaching the right service. Stop the stream at any point to inspect what you have without new rows pushing it down.

## A timeline you can read

The histogram above the table shows log volume over your selected time range, colored by level so errors and warnings stand out from successful requests. A spike in red tells you where to look. Select a range on the timeline to zoom into that window.

## Follow a request through every service

> Click a row to open the detail panel. Instead of a wall of JSON, you see the request as it moved through your project: the network timing, the gateway, the database query, the Auth check. Each service adds its own block with the fields that matter for that service. The raw JSON is still there in a second tab when you need it.

Reviewer: ```suggestion
Click a row to open the detail panel. Instead of a wall of JSON, you see the request as it moved through your project: the network timing, the gateway, the database query, the Auth check. Each service adds its own block with the fields that matter. The raw JSON is still there in a second tab when you need it.
```
